test/test_conditionals.py

```diff
@@ -6,34 +6,46 @@ def test_if(self):
         js_code = 'if (1) {\n\tconsole.log("This is true")\n}'
         py_code = 'if (1):\n\tprint("This is true")'
         bash_code = 'if [[ 1 ]]; then\n\techo "This is true"\nfi'
+        java_code = 'if (1) {\n\tSystem.out.println("This is true");\n}'
         self.assertEqual(py_code, main.main(js_code, 'js', 'py'))
         self.assertEqual(js_code, main.main(py_code, 'py', 'js')) 
         self.assertEqual(bash_code, main.main(py_code, 'py', 'bash'))
         self.assertEqual(bash_code, main.main(js_code, 'js', 'bash'))
+        self.assertEqual(java_code, main.main(js_code, 'js', 'java'))
```

Comment:
These tests are starting to get confusing. I created a [taiga ticket](https://tree.taiga.io/project/jackdavidweber-cjs-capstone/us/148?no-milestone=1) to break up all tests by language and I think this serves as a good example for the need.

Reply:
(To be dealt with in another PR)